Jenkinsfile

```diff
@@ -179,7 +179,7 @@ pipeline {
             catchError(buildResult: null, stageResult: 'FAILURE') {
                 echo '...Cleaning up workspace'
                 cleanWs()
-                sh 'rm -rf ~/.m2/repository'
+                sh 'rm -rf ~/.m2/repository || true'
```

Comment:
`rm -rf` returned an error and that would fail the build. It was so odd that I took a screenshot.
<img width="1506" height="335" alt="Screenshot from 2025-08-27 18-29-34" src="https://github.com/user-attachments/assets/2d1241dd-1eac-45ac-90ff-2a2e6ab6a0b0" />

In the next step after rm -rf, the mesos slave is shutdown. Unless .m2/repository is some kind of persistent store, rm -rf should not be necessary.